lib/login.dart

```diff
@@ -0,0 +1,217 @@
+import 'package:flutter/material.dart';
+
+class MyLogin extends StatefulWidget {
+  const MyLogin({Key? key}) : super(key: key);
+
+  @override
+  State<MyLogin> createState() => _MyLoginState();
+}
+
+class _MyLoginState extends State<MyLogin> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
```

Comment:
Try not to use the AppBar here, as we want the gradient to be visible inside the body as well. Inside the body property, use a Stack to provide two layers, one layer to act as background containing gradient, and the one layer on top of it to display the content. That way, the gradient will be visible.